README.md

```diff
@@ -41,6 +41,12 @@ and improves code composability and readability. More documentation specific to
 
 In addition to GEMMs, CUTLASS implements high-performance convolution via the implicit GEMM algorithm. Implicit GEMM is the formulation of a convolution operation as a GEMM thereby taking advantage of CUTLASS's modular GEMM pipeline. This allows CUTLASS to build convolutions by reusing highly-optimized GEMM components.
 
+## CUTLASS with SYCL
+CUTLASS 3.0 API now also supports SYCL, and can run on Nvidia(upto the Ampere architecture) and Intel Xe Core architecture GPUs using the SYCL backend using the Intel open source `DPC++` compiler.
+The support is currently limited to GEMMs only. See [Quick Start Guide](./media/docs/quickstart.md) on how to build and run
```

Comment:
The documentation on how to build and run with SYCL is in https://github.com/codeplaysoftware/cutlass-fork/blob/sycl-develop/media/docs/build/building_with_sycl_support.md (need to be updated). Would it makes sense to link to that doc instead?

Reply:
Since Nvidia uses and references the quickstart.md above, I decided to add it in quickstart.md.
I did not know what file existed, but I believe that can be deleted now, as many of the things in that file is now being covered by the link to intel/llvm repo, and the remaining in quickstart.md